packages/orbit-components/src/Toast/index.jsx

```diff
@@ -0,0 +1,109 @@
+// @flow
+import * as React from "react";
+import styled, { css } from "styled-components";
+
+import getSpacingToken from "../common/getSpacingToken";
+import mq from "../utils/mediaQuery";
+import Stack from "../Stack";
+import defaultTheme from "../defaultTheme";
+import Text from "../Text";
+import { resolveFadeAnimations, lightAnimation } from "./helpers";
+
+import type { Props } from ".";
+
+const StyledWrapper = styled(({ className, children, dataTest }) => (
+  <div className={className} data-test={dataTest}>
+    {children}
+  </div>
+))`
+  ${({ offsetX, offsetY, theme }) => css`
+    position: absolute;
+    left: 50%;
+    top: ${offsetY};
+    z-index: ${theme.orbit.zIndexOnTheTop};
+    transform: translateX(${offsetX});
+    margin-bottom: ${getSpacingToken};
+  `}
+`;
+
+// $FlowFixMe: https://github.com/flow-typed/flow-typed/issues/3653#issuecomment-568539198
+StyledWrapper.defaultProps = {
+  theme: defaultTheme,
+};
+
+const StyledInnerWrapper = styled.div`
+  ${({ theme, animate }) => css`
+    position: relative;
+    cursor: pointer;
```

Comment:
Why `cursor: pointer`? Nothing happens on click.

Reply:
it's because in the first version it's supposed to be clickable. Plus